lib/locales/it.ts

```diff
@@ -447,6 +454,7 @@ export const locale: Locale = {
     patrick_of_ireland_bishop_patron_of_ireland: 'San Patrizio, vescovo e patrono dell’Irlanda',
     paul_miki_and_companions_martyrs: 'San Paolo Miki sacerdote e compagni, martiri',
     paul_of_the_cross_priest: 'San Paolo della Croce, sacerdote',
+    paul_vi_pope: 'San Paolo VI, papa', // missal_it_2020_ed3
```

Comment:
Just a nitpick: we might change this, however, everywhere I use `mr_${lang}_${year}_${edition}` format.

```suggestion
    paul_vi_pope: 'San Paolo VI, papa', // src: mr_it_2020_ed3
```